airbyte-db/src/main/java/io/airbyte/db/ServerUuid.java

```diff
@@ -39,7 +39,7 @@
   public static Optional<String> get(Database database) throws SQLException {
     return database.query(ctx -> {
       Result<Record> result =
-          ctx.select().from("airbyte_metadata").where(field("key").eq("server-uuid")).fetch();
+          ctx.select().from("airbyte_metadata").where(field("key").eq("server_uuid")).fetch();
```

Comment:
i think the underscore is more idiomatic.

Reply:
Is this a change you are making in the Database to be used as an example test case for a migration too?

Reply:
nope. this was just bugging me. i'll put it in its own PR to avoid confusion.